src/kani-compiler/cbmc/src/goto_program/location.rs

```diff
@@ -13,7 +13,14 @@ pub enum Location {
     BuiltinFunction { function_name: InternedString, line: Option<u64> },
     /// Location in user code.
     /// `function` is `None` for global, `Some(function_name)` for function local.
-    Loc { file: InternedString, function: Option<InternedString>, line: u64, col: Option<u64> },
+    Loc {
```

Comment:
Other than asserts, do we ever want to have `comment` or `property_classes` on statements/expressions. What do we do if there are statements with no property classes?